xbee_remote_at_block.py

```diff
@@ -50,6 +57,21 @@ def _remote_at(self, command, parameter, dest_addr):
         # frame_id is an arbitrary value, 1 hex byte, used to associate sent 
         # packets with their responses. If set to 0 no response will be sent.
         # Could be a block property.
-        self._xbee.send(
-            'remote_at', frame_id=b'\x01', dest_addr=dest_addr,
-            command=command, parameter=parameter)
+        if self.digimesh():
+            self._xbee.send('remote_at',
+                            id=b'\x17',
+                            frame_id=b'\x01',
+                            dest_addr_long=dest_addr,
+                            reserved=b'\xFF\xFE',
+                            options=b'\x02',
+                            command=command,
+                            parameter=parameter)
+        else:
+            self._xbee.send('remote_at',
+                            id=b'\x17',
+                            frame_id=b'\x01',
+                            dest_addr_long=b'\x00\x00\x00\x00\x00\x00\x00\x00',
```

Comment:
Since we don't actually need all the extra stuff when it's not digimesh, I wouldn't include it. Just makes it look more complicated. In fact, I would probably add a comment to the digimesh part and explain why all these params are being used even though they should be defaults.